pycqed/measurement/calibration/single_qubit_gates.py

```diff
@@ -215,16 +215,104 @@ def run_analysis(self, **kw):
 
 
 class ParallelLOSweepExperiment(CalibBuilder):
-    def __init__(self, task_list, sweep_points=None, **kw):
+    """
+    Base class for (parallel) calibration measurements where the LO is swept
+    in sweep dimension 1. The class is based on the concept of a
+    multitasking experiment, see docstrings of MultiTaskingExperiment and
+    of CalibBuilder for general information.
+
+    The following keys in a task are interpreted by this class:
+    - qb: identifies the qubit measured in the task (child classes should be
+        implemented in a way that they adopt this convention)
+    - fluxline: qcodes parameter to adjust the DC flux offset of the qubit.
+        If this is provided, the DC flux offset will be swept together with
+        the frequency sweep such that the qubit is tuned to the current
+        drive frequency (based on the fit_ge_freq_from_dc_offset parameter
+        in the qubit object).
+    - fp_assisted_ro_calib_flux (to be used in combination with fluxline):
+        Specifies the flux (in units of Phi0 with 0 indicating the upper
+        sweep spot) for which the flux-pulse-assisted RO of the qubit has
+        been calibrated. If this is provided and fluxline is provided,
+        the amplitude of the flux pulse assisted RO will be adjusted
+        together with the DC offset during the frequency sweep in order to
+        keep the qubit frequency during RO the same as in the calibration.
+        This required an HDAWG as flux AWG.
+
+    Note: parameters with a (*) have not been exhaustively tested for
+    parallel measurements.
+
+    :param task_list:  see MultiTaskingExperiment
+    :param sweep_points: (SweepPoints object or list of dicts or None)
+        sweep points valid for all tasks.
+    :param allowed_lo_freqs: (list of float or None) (*) if not None,
+        it specifies that the LO should only be set to frequencies in the
+        list, and the desired drive frequency is obtained by an IF sweep
+        using internal modulation of the HDAWG. This requires an HDAWG as
+        drive AWG.
+    :param adapt_drive_amp: (bool) (*) if True, the drive amplitude is adapted
+        for each drive frequency based on the parameter
+        fit_ge_amp180_over_ge_freq of the qubit using the output amplitude
+        scaling of the HDAWG. This requires drive AWG.
+    :param adapt_ro_freq: (bool, default: False) (*) if True, the RO LO
+        frequency is adapted for each drive frequency based on the parameter
+        fit_ro_freq_over_ge_freq of the qubit
+
+    :param kw: keyword arguments.
+        The following kw arguments are interpreted by resolve_lo_sweep_points:
```

Comment:
I thought I'd put it in the class docstring in this case because the average user will never call resolve_lo_sweep_points manually, so they would rather read the class docstring. But I see that it is a bit inconsistent. What do you think?

Reply:
Indeed I would find it useful to list the accepted kwargs in the class docstring but i would have the detailed description in the corresponding method. I give up reading when I see the class docstring is too long :p 
But like i said we can also keep it like this. Up to you.

Reply:
done